doc/cephfs/fs-volumes.rst

```diff
@@ -125,7 +125,8 @@ uid, gid, file mode in octal numerals, and size in bytes. The size of the subvol
 specified by setting a quota on it (see :doc:`/cephfs/quota`). By default a
 subvolume is created within the default subvolume group, and with an octal file
 mode '755', uid of its subvolume group, gid of its subvolume group, data pool layout of
-its parent directory and no size limit.
+its parent directory and no size limit. The subvolume can be created in a separate
+RADOS namespace by sepcifying --isolate-namespace option.
```

Comment:
s/sepcifying/specifying
And add this sentence prior to the sentence starting with 'By default'

Reply:
Done